docs/source/python_tutorials.rst

```diff
@@ -2,8 +2,7 @@ Tutorials
 =========
 
 .. toctree::
-   :maxdepth: 1
-   :caption: Tutorials
+   :maxdepth: 2
```

Comment:
Same feeling here about visual clutter. Do you have a screenshot?

Reply:
Reverted the depth.